julia-mode-tests.el

```diff
@@ -595,6 +595,32 @@ end
 return fact(x)
 end" 'end-of-defun "n == 0" "return fact(x)[ \n]+end" 'end 2))
 
+;;;
+;;; substitution tests
+;;;
+
+(defun julia--substitute (contents position)
+  "Call LaTeX subsitution in a buffer with `contents' at point
+`position', and return the resulting buffer."
+  (with-temp-buffer
+    (julia-mode)
+    (insert contents)
+    (goto-char position)
+    (julia-latexsub)
+    (buffer-string)))
+
+(ert-deftest julia--test-substitutions ()
+  (should (equal (julia--substitute "\\alpha " 7) "α "))
+  (should (equal (julia--substitute "x\\alpha " 8) "xα "))
+  (should (equal (julia--substitute "\\kappa\\alpha(" 13) "\\kappaα("))
+  ; (should (eq (julia--substitute "\\alpha" 7) "α")) ; BROKEN
```

Comment:
Did you use `eq` on purpose here? It indeed fails as is, but passes with `equal`. On the other hand, the next test (with position 6) fails even with `equal`,

Reply:
No, it just a mistake. I updated the code. Thanks!